tests/helpers/helpers.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
```

Comment:
Generic helpers, that are not specific to nydus, and help with making shellscript less painful :)